py-polars/tests/unit/datatypes/test_temporal.py

```diff
@@ -1375,20 +1374,20 @@ def test_replace_time_zone_sortedness_series(
     from_tz: str | None, expected_sortedness: bool, ambiguous: Ambiguous
 ) -> None:
     ser = (
-        pl.Series("ts", [1603584000000000, 1603587600000000])
+        pl.Series("ts", [1603584000000001, 1603587600000000])
```

Comment:
@orlp I think this is what you were asking for? like this, the example shows why the sorted flag shouldn't be preserved for 'Europe/London'